src/layout/Menu.tsx

```diff
@@ -79,17 +79,17 @@ export default function DashboardMenu(): JSX.Element {
 function MenuItem({ item, className }: MenuItemProps): JSX.Element {
   return (
     <li
-      className="group h-12 text-slate-400 hover:bg-cyan-700 hover:text-white hover:w-64 hover:rounded-r-sm"
+      className="group h-12 text-slate-400 lg:hover:bg-cyan-700 lg:hover:text-white lg:hover:w-64 lg:hover:rounded-r-sm"
```

Comment:
Don't show text on hover on small screens as it's weird. We should instead just show the whole text expanded